site/content/en/docs/manual/advanced/filter.md

```diff
@@ -54,6 +58,12 @@ The "Add rule" button adds a rule for objects display. A rule may use the follow
 
 `Between`; `Not between` – these operators allow you to choose a range between two values.
 
+`Like` - this operator indicate that the property must contain a value.
+
+`Starts with`; `Ends with` - filter by beginning or end.
+
+`Proximity search` - looks for text where two separate words are within a specified `near` range.
```

Comment:
Let's skip this item. It was added accidentantly and will be removed in the future